crud.py

```diff
@@ -57,13 +63,28 @@ async def update_market_product(product_id: str, **kwargs) -> Optional[Products]
         (*kwargs.values(), product_id),
     )
     row = await db.fetchone("SELECT * FROM market.products WHERE id = ?", (product_id,))
+    product = Products(**row) if row else None
+    assert product
+
+    stall = await get_market_stall(product.stall)
+    assert stall
+    if stall.currency != "sat" and stall.fiat_base_multiplier:
+        product.price /= stall.fiat_base_multiplier
 
-    return Products(**row) if row else None
+    return product
 
 
 async def get_market_product(product_id: str) -> Optional[Products]:
     row = await db.fetchone("SELECT * FROM market.products WHERE id = ?", (product_id,))
-    return Products(**row) if row else None
+    product = Products(**row) if row else None
```

Comment:
- suggestion: introduce `Product.from_row(row, currency, fiat_base_multiplier)`
- have this logic in one place alone